### PR TITLE
geowave/config/addindex and geowave/config/copyindex

### PR DESCRIPTION
**Pull request by mrkev**
👍

### CODE REVIEW

File: core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/AddIndexCommand.java [hunk 1]
```diff
@@ -177,9 +148,78 @@ public void setPluginOptions(
 	}
 
 	@Override
+	public void readFormArgs(
```
Comment (mak438) [inline core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/AddIndexCommand.java]: Remove. DefaultOperation reads the form args automatically now.

File: core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/CopyIndexCommand.java [hunk 2]
```diff
@@ -104,12 +104,12 @@ public Void computeResults(
 	@Override
 	public void readFormArgs(
```
Comment (mak438) [inline core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/CopyIndexCommand.java]: Remove. DefaultOperation reads the form args automatically now.

File: core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/remote/options/IndexPluginOptions.java [hunk 3]
```diff
@@ -90,18 +90,35 @@ public String getType() {
 		return indexType;
 	}
 
+	// public void setType(String type) { this.indexType = type; }
+
 	public int getNumPartitions() {
 		return numPartitions;
 	}
 
+	public void setNumPartitions(
```
Comment (mak438) [inline core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/remote/options/IndexPluginOptions.java]: I don't see where these are used. Are they needed?

File: services/rest/src/test/java/mil/nga/giat/geowave/service/rest/RestServerTest.java [hunk 4]
```diff
@@ -250,11 +250,108 @@ public void geowave_config_store()
 	}
 
 	// Tests geowave/config/addindex, cpindex, rmindex
-	@Ignore("Kevin will fix addindex")
+	// @Ignore("Kevin will fix addindex")
```
Comment (mak438) [inline services/rest/src/test/java/mil/nga/giat/geowave/service/rest/RestServerTest.java]: Remove this line.

**Review by mak438 — CHANGES_REQUESTED** [review]

**Review by mak438 — APPROVED** [review]

**Comment by mak438** [comment]
Please squash if you can.

**Comment by mrkev** [comment]
@mak438 Github can squash automatically when you merge a PR, it's one of the options of the merge button

**Comment by mrkev** [comment]
If you give me a thumbs up ill just squash and merge now

**Comment by mak438** [comment]
Go for it! :thumbsup:

**Pull request merged**